content/influxdb3/cloud-dedicated/reference/admin-ui.md

```diff
@@ -12,89 +12,174 @@ weight: 105
 The Administrative (Admin) UI for {{% product-name %}} is a browser-based, no-code way to manage your {{% product-name %}} environment and perform administrative tasks, such as creating and managing clusters, databases, and tokens.
 
 - [Access the Admin UI](#access-the-admin-ui)
-- [Account management](#account-management)
+- [Navigate the Admin UI](#navigate-the-admin-ui)
+- [Account and cluster management](#account-and-cluster-management)
 - [Resource management](#resource-management)
-  - [Manage clusters](#manage-clusters)
+  - [View cluster overview and metrics](#view-cluster-overview-and-metrics)
   - [Manage databases](#manage-databases)
   - [Manage tables](#manage-tables)
   - [Manage database tokens](#manage-database-tokens)
-  - [Additional Features](#additional-features)
-
+  - [Manage management tokens](#manage-management-tokens)
```

Comment:
The follow links don't look to work for me (I don't think they work in the existing docs either). They look to point to anchors that don't exist.

View cluster information
Access operational dashboards
Administer management tokens